introduce two variants of QuicTLS

chipitsine opened the pull request:
previously QuicTLS was a patchset on top of OpenSSL, let us name it as OldQuicTLS, currently QuicTLS is not bound to OpenSSL, it will be named QuicTLS

a-denoyelle commented:
I'm fine with the change, the only thing is do not really like the name OldQuicTLS. Maybe we can use something as `QuicTLS-1.1.1` to highlight that it is still based on the old openssl branch ?

chipitsine committed:
introduce two variants of QuicTLS

previously QuicTLS was a patchset on top of OpenSSL, let us name it as
OldQuicTLS, currently QuicTLS is not bound to OpenSSL, it will be named
QuicTLS

a-denoyelle commented:
Merged, thanks.
